test/amp_tags_test.js

```diff
@@ -51,7 +51,7 @@ describe('AmpTags', function() {
   })
 
   it("does append domain to amp stories if appendHostToAmpUrl present", function () {
-    const story = {"slug": "section/slug", "is-amp-supported": true}
+    const story = {"slug": "section/slug", "is-amp-supported": true, "url": "https://madrid.quintype.io/section/slug" }
     const string = getSeoMetadata({...seoConfig, appendHostToAmpUrl: true}, config, 'story-page', {data: {story: story}}, {})
```

Comment:
Should we test with an invalid URL?
let's say in future the URL part is removed and we don't want this to be broken. I know it's an API result. but still.